basic_crud/basic_crud/basic_crud.py

```diff
@@ -0,0 +1,202 @@
+"""Welcome to Reflex! This file outlines the steps to create a basic app."""
+import asyncio
+import json
+from basic_crud.api import product_router
+from basic_crud.model import Product
```

Comment:
```suggestion
from .api import product_router
from .model import Product
```

using relative import makes the example easier to copy/paste into another project